cmd/eksctl-anywhere/cmd/upgrademanagementcomponents.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/aws/eks-anywhere/cmd/eksctl-anywhere/cmd/aflag"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/types"
+	"github.com/aws/eks-anywhere/pkg/workflows/management"
+	"github.com/spf13/cobra"
+)
+
+type upgradeManagementComponentsOptions struct {
+	clusterOptions
+}
+
+var umco = &upgradeManagementComponentsOptions{}
+
+func init() {
+	flagSet := upgradeManagementComponentsCmd.Flags()
+	aflag.String(aflag.ClusterConfig, &umco.fileName, flagSet)
+	aflag.String(aflag.BundleOverride, &umco.bundlesOverride, flagSet)
+}
+
+var upgradeManagementComponentsCmd = &cobra.Command{
+	Use:          "management-components",
+	Short:        "Upgrade management components in a management cluster",
+	Long:         "This command is used to upgrade management components in a management cluster",
```

Comment:
This long description would be a great place to link to or answer questions like: What are the names of the management components? Generally speaking, what will be involved in this process? What is the expected impact to management and workload clusters during this process? Are there any things to take into consideration for when i should or shouldnt run this upgrade process? etc

I don't think all of these questions should or need to be answered in this long description but there is a real opportunity to provide actual value and confidence to customers in this long description.